### PR TITLE
docs: edits and updates

### PR DESCRIPTION
**Pull request by huayuenh**
### Description

Replace this text with a summary of the changes in this PR. Include why the changes are needed and context about the changes. List required dependencies. If there is a Git issue for the change, please link to it.

#### No release required

- [ ] Examples or tests (addition or updates of examples or tests)
- [x] Documentation update
- [ ] CI-related update (pipeline, etc.)
- [ ] Other changes that don't affect Terraform code

#### Release required

- [ ] Bug fix (patch release (`x.x.X`): Change that fixes an issue and is compatible with earlier versions)
- [ ] New feature (minor release (`x.X.x`): Change that adds functionality and is compatible with earlier versions)
- [ ] Breaking change (major release (`X.x.x`): Change that is likely incompatible with previous versions)

##### Release notes content

Replace this text with information that users need to know about the bug fixes, features, and breaking changes. This information helps the merger write the commit message that is published in the release notes for the module.

---

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- Merge by using "Squash and merge".
- Use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author.

    The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).

### CODE REVIEW

**Review by christoc-ibm — APPROVED** [review]

File: reference-architectures/deploy-arch-ibm-devsecops-alm.md [hunk 1]
```diff
@@ -28,30 +28,22 @@ use-case: DevOps
...
 {: toc-use-case="DevOps"}
-{: toc-compliance=""}
 
-The DevSecOps deployable architecture creates a set of DevOps Toolchains and pipelines. DevSecOps uses Continuous Delivery (Git Repos and Issue Tracking, Tekton Pipelines,
-DevOps Insights, and Code Risk Analyzer), Secrets Manager, Key Protect, Cloud Object Storage, Container Registry and Vulnerability Advisor. Out of the box, DevSecOps also
-leverages popular scanning tools such as SonarQube, GoSec, OWASP Zap (dynamic scan), any unit test framework, and GPG signing. It can also be used with more tools such as
-external Git providers and artifact stores. DevSecOps supports hybrid deployments, in particular by using private pipeline workers, and can be interfaced with other
-deployment tools such as Satellite Config.
+The DevSecOps deployable architecture creates a set of DevOps Toolchains and pipelines. DevSecOps uses Continuous Delivery (Git Repos and Issue Tracking, Tekton Pipelines, DevOps Insights, and Code Risk Analyzer), Secrets Manager, Key Protect, Cloud Object Storage, Container Registry and Vulnerability Advisor.
+{: shortdesc}
```
Comment (gmarjoram) [inline reference-architectures/deploy-arch-ibm-devsecops-alm.md:44]: What's the {:shortdesc} here for .... a title for the next paragraph ?

**Review by gmarjoram — APPROVED** [review]
One question but otherwise LGTM

File: reference-architectures/deploy-arch-ibm-devsecops-alm.md (hunk 1 above)
Comment (christoc-ibm) [reply reference-architectures/deploy-arch-ibm-devsecops-alm.md:44]: @gmarjoram - It's markup that's required for IBM Cloud docs. It identifies the preceding paragraph as the content that's included in SEO. Everything that Jenifer changed in this topic was to make the content follow the markup and language guidelines. I hope this helps!

Comment (gmarjoram) [reply reference-architectures/deploy-arch-ibm-devsecops-alm.md:44]: Thanks ... it does 👍

**Pull request merged**